src/pybert/pybert.py

```diff
@@ -152,14 +105,14 @@ class PyBERT(HasTraits):
         default_value="PRBS-7",
     )
     seed = Int(1)  # LFSR seed. 0 means regenerate bits, using a new random seed, each run.
-    nspb = Range(low=2, high=256, value=gNspb)  #: Signal vector samples per bit.
-    eye_bits = Int(gNbits // 5)  #: # of bits used to form eye. (Default = last 20%)
-    mod_type = List([0])  #: 0 = NRZ; 1 = Duo-binary; 2 = PAM-4
-    num_sweeps = Int(1)  #: Number of sweeps to run.
-    sweep_num = Int(1)
-    sweep_aves = Int(gNumAve)
-    do_sweep = Bool(False)  #: Run sweeps? (Default = False)
-    debug = Bool(False)  #: Send log messages to terminal, as well as console, when True. (Default = False)
+    nspb = Range(low=2, high=256, value=32)  #: Signal vector samples per bit.
+    mod_type   = List([0])                   #: 0 = NRZ; 1 = Duo-binary; 2 = PAM-4
+    num_sweeps = Int(1)                      #: Number of sweeps to run.
+    sweep_num  = Int(1)
+    sweep_aves = Int(1)
+    do_sweep   = Bool(False)  #: Run sweeps? (Default = False)
+    debug      = Bool(False)  #: Send log messages to terminal, as well as console, when True. (Default = False)
+    rel_thresh = Float(3.0)   #: Spectral threshold for identifying periodic components (sigma). (Default = 3.0)
```

Comment:
Renaming the attributes that get saved into the config such as thresh -> rel_thresh breaks the current loading/saving implementation.  Need to make the same change here: https://github.com/capn-freako/PyBERT/blob/master/src/pybert/configuration.py#L151

Reply:
I'm okay with the rename just another note that we don't have any way to update our configuration to a new format yet; so will break older configurations.

Reply:
Whoops! I'll change that back to `thresh`.
Were there any others?

Reply:
I think just the one attribute was changed.